scripts/update.sh

```diff
@@ -71,28 +72,30 @@ parse_args() {
 }
 
 setup() {
-  if [[ ! -d $OXA_PATH/oxa-tools-config ]]; then
+  if [[ ! -d $OXA_ENV_SETTINGS_PATH ]]; then
     cd $OXA_PATH
 
     # Fetch the latest secrets from the private repo via a personal access token
+    # this will create $OXA_ENV_SETTINGS_PATH directory.
     sudo git clone -b master https://$ACCESS_TOKEN@github.com/microsoft/oxa-tools-config.git
   fi
-  source $OXA_PATH/oxa-tools-config/env/$DEPLOYMENT_ENV/$DEPLOYMENT_ENV.sh
+  source $OXA_ENV_SETTINGS_PATH/env/$DEPLOYMENT_ENV/$DEPLOYMENT_ENV.sh
   export CONFIGURATION_REPO CONFIGURATION_VERSION
 
   if [[ ! -d $OXA_CONFIG_PATH ]]; then
-    wget https://raw.githubusercontent.com/edx/configuration/master/util/install/ansible-bootstrap.sh -O - | bash
-
-    git clone $CONFIGURATION_REPO $OXA_CONFIG_PATH
+    git clone -b $CONFIGURATION_VERSION $CONFIGURATION_REPO $OXA_CONFIG_PATH
     cd $OXA_CONFIG_PATH
-    git checkout $CONFIGURATION_VERSION
 
```

Comment:
good point. these changes are cleanup I wanted to split off from functional changes. i will incorporate this into an upcoming pull request